package.json

```diff
@@ -20,8 +20,9 @@
     "@langchain/community": "0.0.56",
     "@langchain/core": "0.1.63",
     "@langchain/groq": "0.0.9",
-    "@langchain/langgraph": "0.0.12",
+    "@langchain/langgraph": "0.0.23",
     "@langchain/openai": "0.0.29",
+    "@supabase/supabase-js": "^2.43.5",
```

Comment:
Can you try to add 

  "overrides": {
    "@langchain/core": "0.2.5"
  }
  below are dependencies for consistency